.env.development

```
@@ -5,3 +5,5 @@
 # DO NOT put secrets in this file - use .env.development.local instead
 
 NEXTJS_ENV=development
+NEXT_PUBLIC_SITE_URL=http://localhost:3000
+ANALYTICS_HASH_SECRET=1f36b08c94c21f9916f3adb62248331243c0021545a9f3c337e049eba9db3a50
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ANALYTICS_HASH_SECRET key should go before the NEXTJS_ENV key

(UnorderedKey)